src/replica/storage/simple_kv/test/case-209.act

```diff
@@ -59,96 +59,95 @@ client:replica_config:receiver=r1,type=downgrade_to_inactive,node=r2
 config:{3,r1,[]}
 state:{{r1,pri,3,10}}
 
+# add secondary r3 (new replica)
+client:replica_config:receiver=r1,type=add_secondary,node=r3
+config:{4,r1,[r3]}
+state:{{r1,pri,4,10},{r3,sec,4,10}}
+
+# write #11 - #15
+client:begin_write:id=11,key=k11,value=v11,timeout=0
+client:end_write:id=11,key=k11,err=err_ok,resp=0
+client:begin_write:id=12,key=k12,value=v12,timeout=0
+client:end_write:id=12,key=k12,err=err_ok,resp=0
+client:begin_write:id=13,key=k13,value=v13,timeout=0
+client:end_write:id=13,key=k13,err=err_ok,resp=0
+client:begin_write:id=14,key=k14,value=v14,timeout=0
+client:end_write:id=14,key=k14,err=err_ok,resp=0
+client:begin_write:id=15,key=k15,value=v15,timeout=0
+client:end_write:id=15,key=k15,err=err_ok,resp=0
+
+state:{{r1,pri,4,15,15},{r3,sec,4,15,15}}
+
+# remove secondary r3
+client:replica_config:receiver=r1,type=downgrade_to_inactive,node=r3
+config:{5,r1,[]}
+state:{{r1,pri,5,15}}
+
 # remove primary r1
 client:replica_config:receiver=r1,type=downgrade_to_inactive,node=r1
-config:{4,-,[]}
+config:{6,-,[]}
 state:{}
 
-# assign primary r3
-client:replica_config:receiver=r3,type=assign_primary,node=r3
-config:{5,r3,[]}
-state:{{r3,pri,5,0}}
-
-# add secondary r2
-# will trigger learning in on_learn():
-#   _prepare_list.count() {=0} == 0 
-#   learn_start_decree {=11} > last_prepared_decree(0)
-#   learner state is newer than learnee, learn from scratch
-#
-#                      (0)
-#   prepare_list :      ||
-#   learn        :                           |-->
-#                                           (11)
-#
-#   set:simple_kv_close_fail=1 to inject failure when reopen the app
```

Comment:
Same suggestion, I suggest remaining this comment to show the failure injection.